db/schemas/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .schemas import AssessmentRecordMetadata
+from .schemas import AssessorTaskListMetadata
 
-__all__ = [AssessmentRecordMetadata]
+__all__ = ["AssessmentRecordMetadata", "AssessorTaskListMetadata"]
```

Comment:
💯

Reply:
just out of interest folks, is good practice pythonista stuff? My understanding was that the default 'import *' behaviour is to import all symbols that do not begin with an underscore anyway?

Reply:
__all__ specifies what happens when you use *. While we don't use * often, many libraries do (like pdoc) and it keeps the public facing parts very clear. It's also used by pdoc to figure out what are the python things worth documenting.